config.yaml

```diff
@@ -1,88 +1,85 @@
-common:
- work_dir: 'workspace'
- result_dir: 'result'
- verbose: False
- log_size: 2097152
- packages_metadata_repo_url: "https://github.com/open-power-host-os/versions.git"
- packages_metadata_repo_branch: 'master'
- http_proxy: ''
- distro_name: 'CentOS'
- distro_version: '7'
- arch_and_endianness: 'ppc64le'
- mock_binary: "/usr/bin/mock"
- commit_updates: True
- push_updates: True
- updater_name:
- updater_email:
+build_iso:
+  automated_install_file: host-os.ks
+  automated_install_packages_groups:
+  - '@base'
```

Comment:
This is the default serialization behavior in PyYAML and yamllint does not complain.